src/javascripts/Services/hotkeyService.js

```diff
@@ -33,19 +33,30 @@ ngapp.service('hotkeyService', function(hotkeyFactory) {
             if (!action) return;
             action = action.callback;
         }
-        action.Constructor === Function ? action(scope, e) : scope[action](e);
+        typeStr === 'function' ? action(scope, e) : scope[action](e);
         e.stopImmediatePropagation();
         e.preventDefault();
     };
 
-    this.buildOnKeyDown = function(scope, label, view) {
-        let hotkeys = hotkeyFactory[`${view}Hotkeys`];
-        scope[label] = function(e) {
+    let keyEventHandler = function(scope,hotkeys,type) {
```

Comment:
:rocket: need a space between `scope,` and `hotkeys`.  also need a space between `hotkeys,` and `type`